aep/general/0134/aep.md.j2

```diff
@@ -0,0 +1,321 @@
+# Update
+
+In REST APIs, it is customary to make a `PATCH` or `PUT` request to a
+resource's URI (for example, `/v1/publishers/{publisher}/books/{book}`) in
+order to update that resource.
+
+Resource-oriented design (AEP-121) honors this pattern through the `Update`
+method (which mirrors the REST `PATCH` behavior). These RPCs accept the URI
+representing that resource and return the resource.
+
+## Guidance
+
+APIs **should** provide an update method for resources unless it is not
+valuable for users to do so. The purpose of the update method is to make
+changes to the resources without causing side effects.
+
+Update methods are specified using the following pattern:
+
+- The RPC's name **must** begin with the word `Update`. The remainder of the
+  RPC name **should** be the singular form of the resource's message name.
+- The request message **must** exactly match the RPC name, with a `Request`
+  suffix.
+- The response message **must** be the resource itself.
+  - The response **should** include the fully-populated resource, and **must**
+    include any fields that were sent and included in the update mask unless
+    they are input only (see AEP-203).
+  - If the update RPC is [long-running](#long-running-update), the response
+    message **must** be an `Operation` for which the return type is the
+    resource itself.
+- The method **should** support partial resource update, and the HTTP verb
+  **should** be `PATCH`.
```

Comment:
This is all porto-specific guidance, because it uses "RPC" and "message". We should either rephrase to be generic -- perhaps "RPC" -> "method" and "message" -> "content" -- or move this to the proto section and provide equivalent guidance in the oas section.

Reply:
+1 - I think we should normalize the structure of the AEPs to always include protocol guidance *after* examples - I think that structure wasn't always consistent.

Reply:
Done.